PSKoans/Koans/Foundations/AboutStrings.Koans.ps1

```diff
@@ -47,13 +47,12 @@ Describe 'Strings' {
         }
 
         It 'can do a simple expansion' {
-            $Windows = Get-Item 'C:\Windows' | Select-Object -ExpandProperty FullName
-            '__' | Should -Be "The windows directory is located here: $Windows"
+            '__' | Should -Be "Your home directory is located here: $HOME"
         }
 
         It 'handles other ways of doing the same thing' {
             # Strings can handle entire subexpressions being inserted as well!
-            $String = "The windows directory is located at $(Get-Item 'C:\Windows')"
+            $String = "Your default profile exists?: $(Test-Path $PROFILE)"
```

Comment:
```suggestion
            $String = "Your PowerShell profile path is '$PROFILE'"
```

If only because many people new to PowerShell may not be aware of profiles, do this koan, and then later create one... and suddenly this koan would fail again. 😄

Reply:
Does that demonstrate "entire subexpressions" being inserted as well?  Or isn't that essentially the same as the previous example of "simple expansion"?  How about something like:

```PowerShell
"You are running the $(Get-Module -Name PSKoans) module!"
```

Reply:
Hmm, you're right. That would probably work. However, if the user has more than one module of that name it will come back with `System.Object[]` 😄 

Perhaps something more like the original, just cross-platform?
```suggestion
            $String = "Your default profile exists?: $(Get-Item $HOME)"
```